master/buildbot/changes/gitpoller.py

```diff
@@ -159,8 +170,16 @@ def poll(self):
             '+%s:%s' % (self._removeHeads(branch), self._trackerBranch(branch))
             for branch in branches
         ]
-        yield self._dovccmd('fetch',
```

Comment:
This would be great -- I think you'll want `log.msg(e.args[0])`, but I'd like to see the resulting log message in the test to be sure.